projects/plugins/jetpack/package.json

```diff
@@ -165,6 +165,7 @@
 		"@testing-library/react": "11.2.7",
 		"@testing-library/react-hooks": "4.0.1",
 		"@testing-library/user-event": "12.8.3",
+		"@wordpress/babel-preset-default": "6.2.0",
```

Comment:
This has been added in a separate PR.

Reply:
Specifically, https://github.com/Automattic/jetpack/pull/20634.